example/index.html

```diff
@@ -3,8 +3,8 @@
 <head>
   <meta http-equiv="content-type" content="text/html; charset=UTF-8">
   <title>Leaflet.Spin</title>
-  <link rel="stylesheet" href="leaflet/dist/leaflet.css">
-  <script src="leaflet/dist/leaflet.js" charset="utf-8"></script>
```

Comment:
As example should be working without `node_modules` available *(to work in [Github Page](http://makinacorpus.github.io/Leaflet.Spin/)*, you can't use Leaflet from dependencies.

So the solution is, for the example page, calling Leaflet from a CDN (as I've done here: https://github.com/makinacorpus/Leaflet.MeasureControl/commit/fac37b2a6e5c56e24d577eb6a9e6c38e55ce4bf6).